### PR TITLE
Switch back to nested table

### PR DESCRIPTION
**Pull request by kjain110**

### CODE REVIEW

File: src/main/resources/entity-types/inventory/simple_instance.json5 [hunk 1]
```diff
@@ -192,18 +192,6 @@
       filterValueGetter: "lower(${tenant_id}_mod_inventory_storage.f_unaccent(:sourceAlias.jsonb ->> 'hrid'::text))",
       valueFunction: 'lower(${tenant_id}_mod_inventory_storage.f_unaccent(:value))',
     },
-    {
-      name: 'match_key',
```
Comment (kjain110) [inline src/main/resources/entity-types/inventory/simple_instance.json5:196]: Kathleen wanted to remove this

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ mod-fqm-manager:
   max-query-size: ${MAX_QUERY_SIZE:1250000}
   bypass-permissions: false # local development ONLY
   permissions-cache-timeout-seconds: 60
-  entity-type-cache-timeout-seconds: 3600
+  entity-type-cache-timeout-seconds: 3
```
Comment (mweaver-ebsco) [inline src/main/resources/application.yml:6]: Should this be here? 3 seconds seems too low for prod

Comment (kjain110) [reply src/main/resources/application.yml:6]: ummm I forgot to change :(

Comment (kjain110) [reply src/main/resources/application.yml]: @mweaver-ebsco , Done

**Review by ncovercash — APPROVED** [review]
ticket #?

**Review by bvsharp — APPROVED** [review]

**Pull request merged**